themes/bootstrap5/js/searchbox_controls.js

```diff
@@ -59,9 +59,10 @@ VuFind.register('searchbox_controls', function SearchboxControls() {
 
   function _updateKeyboardLayout(layoutName) {
     $('.keyboard-selection-item').each(function deactivateItems() {
-      $(this).parent().removeClass("active");
+      $(this).removeClass("active");
```

Comment:
I just spoke with @EreMaijala, and he suggested that it might be worth using the [getBootstrapMajorVersion](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap5/js/common.js#L449) function here to conditionally do the right thing for each Bootstrap version, instead of creating an exception for this file in build.xml, to reduce the chances of themes getting out of sync. It should be quick and easy to clean up/simplify once we drop bootstrap3 support. What do you think?

Reply:
Yes, that makes much more sense. Done.